locksmith/src/controllers/v2/hooksController.ts

```diff
@@ -51,8 +54,12 @@ export const guildHook: RequestHandler = async (request, response) => {
 
   const accesses = await Promise.all(
     recipients.map(async (recipient: string) => {
-      const roles = await guild.getUserAccess(hookGuildId, recipient)
-      const hasAtLeastOne = roles.some((role) => role.access)
+      const { user: userClient } = guildClient
+      const userMemberships = await userClient.getMemberships(recipient)
```

Comment:
Does this paginate?

Reply:
Please check if the API will return _all_ of a user's guilds, even if they have a LOT!